scipy/stats/mstats_basic.py

```diff
@@ -87,6 +87,35 @@ def _chk_size(a, b):
     return (a, b, na)
 
 
+def _contains_nan(a, nan_policy='propagate'):
```

Comment:
Best avoid code duplication here. Can you move it from stats.py to mstats and import it in stats.py instead?

Reply:
Yes, I'll do this. Thank you.

Reply:
I believe I have resolved this as you described. Please let me know if I misunderstood what you had in mind. 

I want to make sure to avoid code duplication but I also tried to update the imports so as to avoid having to touch too many other lines of code.